duckduckgo/iOS
# Autofill sorting fix for domains that are exact match to those in tlds.json

Task/Issue URL: https://app.asana.com/0/0/1204425262115661/f
Tech Design URL:
CC: 

**Description**:
Fixes issue where domains that are exact match to a domain contained in tlds.json were sorted under #, instead of by the first character of the domain name



**Steps to test this PR**:
1. Add Autofill Login for url `github.io`, leaving the title blank
2. Confirm it is grouped under `G`
3. Add Autofill Login for url `mysite.github.io`, leaving the title blank
4. Confirm it is grouped under `M`
5. Add Autofill Login for url `example.com`, leaving the title blank
6. Confirm it is grouped under `E`
7. Add Autofill Login for url `sub.example.com`, leaving the title blank
8. Confirm it is grouped under `E`



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
